.gitignore

```diff
@@ -2,3 +2,5 @@
 .gradle
 
 build
+
+!gradle/wrapper/gradle-wrapper.jar
```

Comment:
I don't see how this would be necessary. This is never excluded. We exclude `.gradle`, not `gradle` (mind the dot).

Reply:
Might have been some global gitignore on my system that required that. Removed it again.